Purchases/Error Handling/ErrorCode.swift

```diff
@@ -151,6 +152,8 @@ extension ErrorCode: DescribableError {
             return "SKProductsRequest took too long to complete."
         case .apiEndpointBlockedError:
             return "Requests to RevenueCat are being blocked. See: https://rev.cat/dnsBlocking for more info."
+        case .invalidPromotionalOfferError:
```

Comment:
@aboedo do we have a short link for promotional offers?

Reply:
we do now! https://rev.cat/ios-subscription-offers